features/exploredash/src/main/res/layout/dialog_dash_direct_login_info.xml

```diff
@@ -0,0 +1,97 @@
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
+
+    <ImageButton
+        android:id="@+id/collapse_button"
+        style="@style/Button.Tertiary.ExtraSmall"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end"
+        android:layout_marginTop="20dp"
+        android:paddingHorizontal="0dp"
+        android:src="@drawable/ic_close"
+        android:text="@string/explore_apply_filter" />
+
+    <ImageView
+        android:id="@+id/dialog_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="50dp"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginHorizontal="20dp"
+        tools:src="@drawable/ic_dashdirect_logo" />
+
+    <TextView
+        android:id="@+id/dialog_title"
+        style="@style/Headline5"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="20dp"
+        android:layout_marginTop="20dp"
+        android:gravity="center"
+        android:textAlignment="gravity"
+        android:textStyle="bold"
+        tools:text="@string/buy_a_gift_card_with_your_dash_direct_account" />
+
+    <TextView
+        android:id="@+id/dialog_message"
+        style="@style/Body2"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginHorizontal="20dp"
+        android:layout_marginTop="10dp"
+        android:layout_weight="1"
+        android:gravity="center"
+        android:textAlignment="gravity"
+        app:layout_constraintTop_toTopOf="@id/title"
+        tools:text="@string/create_an_account_at_dash_direct_or_log_in_to_the_existing_one" />
+
+    <Button
+        android:id="@+id/dialog_extra_message"
+        style="@style/Button.Tertiary.Small.Blue"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp"
+        android:layout_marginBottom="25dp"
+        app:layout_constraintTop_toTopOf="@id/title"
+        tools:text="Back up recovery phase" />
```

Comment:
I don't see an icon on this button like in the design. Since you're using a custom layout, you can modify it as necessary and not use the "extra message" pattern.

Reply:
nice catch Fixed